vendor/assets/javascripts/jquery.pageless.js

```diff
@@ -145,13 +145,24 @@
 
   // distance to end of the container
   var distanceToBottom = function () {
-    return (container === window)
-         ? $(document).height()
-         - $container.scrollTop()
-         - $container.height()
-         : $container[0].scrollHeight
-         - $container.scrollTop()
-         - $container.height();
```

Comment:
This has assumed that the _result container_ is at the end of the page. (i.e. it doesn't take into account _result container's_ position or height)